pkg/interface/chat/src/js/components/new.js

```diff
@@ -220,19 +228,29 @@ export class NewScreen extends Component {
         <h2 className="mb3 f8">New Chat</h2>
         <div className="w-100">
           <p className="f8 mt3 lh-copy db">Name</p>
-          <p className="f9 gray2 db mb2 pt1">
-            Lowercase alphanumeric characters, dashes, and slashes only
+          <textarea
+            className={idClasses}
+            placeholder="Secret Chat"
+            rows={1}
+            style={{
+              resize: "none"
+            }}
+            onChange={this.titleChange}
+          />
+              {idErrElem}
+          <p className="f8 mt3 lh-copy db">
+            Description
+            <span className="gray3"> (Optional)</span>
           </p>
           <textarea
             className={idClasses}
-            placeholder="secret-chat"
+            placeholder="The coolest chat"
             rows={1}
             style={{
               resize: "none"
             }}
-            onChange={this.idChange}
+            onChange={this.titleChange}
```

Comment:
Oh, I found one. This should be descriptionChange, ha.

Reply:
good catch, thanks!